### PR TITLE
Change name of electrical series to follow best practices

### PR DESCRIPTION
**Pull request by h-mayorquin**
As described on the title. The issue of whether to capitalize LFP or Lfp in the camel case is still pending.

### CODE REVIEW

**Comment by h-mayorquin** [comment]
I don't have time to add the tests for the edge cases that using a single series name implies. Maybe let's keep this that changes the adapts the old behavior to best practices and then change that later.

**Comment by CodyCBakerPhD** [comment]
> I don't have time to add the tests for the edge cases that using a single series name implies. Maybe let's keep this that changes the adapts the old behavior to best practices and then change that later.

Yeah it's not a huge priority either way

**Comment by h-mayorquin** [comment]
> > I don't have time to add the tests for the edge cases that using a single series name implies. Maybe let's keep this that changes the adapts the old behavior to best practices and then change that later.
> 
> Yeah it's not a huge priority either way

Oh, OK, I was thinking you wanted this for the next release. I just meant that I did not have time to do it last Friday. Let's discuss the final points on our meeting tomorrow and get it done : )

File: requirements-testing.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ pytest
 pytest-cov
 ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow NWBFile load at all timess
 parameterized>=0.8.1
+pre-commit >= 2.20.0
```
Comment (CodyCBakerPhD) [inline requirements-testing.txt:5]: Why is this needed?

Comment (h-mayorquin) [reply requirements-testing.txt:5]: It is not. I added it because it was a development tool that we use but don't have anywhere else when I was installing. It slipped in this PR by mistake though. Feel free to remove it.

Comment (CodyCBakerPhD) [reply requirements-testing.txt:5]: We harness the pre-commit bot at the end of the day for this, but people are always free to install and use locally (but not a part of package or even required of test/dev installations)

**Review by CodyCBakerPhD — DISMISSED** [review]

**Review by CodyCBakerPhD — APPROVED** [review]
